mempool/src/counters.rs

```diff
@@ -409,20 +409,6 @@ pub static DB_ERROR: Lazy<IntCounter> = Lazy::new(|| {
     .unwrap()
 });
 
-/// Gauge for the preference ranking of the current chosen upstream network
-/// to broadcast to
-static UPSTREAM_NETWORK: Lazy<IntGauge> = Lazy::new(|| {
```

Comment:
The only annoying thing about removing counters is having to remember to update the dashboards 😄 We'll probably need to audit a bunch of these components at the end of the half (state sync, included) :P

Reply:
Yeah, I agree.  Maybe I'll make an issue that keeps track of these removals

Reply:
Tracking here: https://github.com/diem/diem/issues/9343 I'll follow up afterwards